### ISSUE TITLE
recurring payments

### ISSUE THREAD

**Issue by wenxinwilliam** [mirumee/django-payments#59]
Do you guys have any plans to support recurring payments?

**Comment by patrys** [comment]
We certainly want to support them but we have not yet decided on an API for subscriptions.

**Comment by jgeskens** [comment]
At my company we have a solution for payment subscriptions, and are thinking about integrating this in django-payments. How do you guys feel about this? Our current solution is closed-source but already used for years in production in a telecom ecommerce website. Apart from that, we also have solutions for payment reconciliation (the possibility to upload a CSV and compare payments, to detect missing payments etc).

**Comment by patrys** [comment]
We are happy to accept a pull request as long as you are happy with the resulting licence being open.

**Comment by jgeskens** [comment]
Of course, that is the whole point of open source :-) Knowing this, I will start looking deeper into the possibilities now. I will keep you guys posted. Thanks!

**Comment by madisvain** [comment]
Has there been any update on this topic? We are looking for the same solution - I'm wondering what payment method were you using?

**Comment by patrys** [comment]
We still don't have a generic solution for django-payments. Unfortunately how different payment gateway handle recurring payments and subscriptions is different enough that building a common abstraction layer is a non-trivial exercise.

**Comment by oesah** [comment]
That's the only important thing missing in my view. Any progress on this issue? I am thinking on using this module for a client, but we need recurring payments for that.

**Comment by pije76** [comment]
Any progress on this issue?

**Comment by kabaltech** [comment]
Any update on this? to support saving cards for pre-auth billing as well

**Comment by tetrahed** [comment]
Are there any updates for this?
